avocado/plugins/remote.py

```diff
@@ -68,10 +68,10 @@ def configure(self, parser):
                                         help='Specify an identity file with '
                                         'a private key instead of a password '
                                         '(Example: .pem files from Amazon EC2)')
-        self.remote_parser.add_argument('--remote-no-copy',
-                                        dest='remote_no_copy',
+        self.remote_parser.add_argument('--remote-copy',
+                                        dest='remote_copy',
                                         action='store_true',
-                                        help="Don't copy tests and use the "
+                                        help="Copy tests and use the "
```

Comment:
OK, I was reluctant about removing them to not impact users that much, but since we are going to change the behaviour anyway, let's remove them once for all.